### PR TITLE
CPLAT-9406 Create Simple Props / State Boilerplate Migrator

### PR DESCRIPTION
**Pull request by joebingham-wk**
## Motivation
  
We need a migrator that just takes the most basic props and state classes and turns them into mixins.
## Changes
  
- Built out the `SimplePropsAndStateClassMigrator`
- Add tests
## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        - [ ] CI passes
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Client Platform member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react_codemod/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/ignoreable.dart [hunk 1]
```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 import 'package:codemod/codemod.dart';
-import 'package:path/path.dart' as path;
```
Comment (joebingham-wk) [inline lib/src/ignoreable.dart]: Just cleaning up a lint

File: test/boilerplate_suggestors/props_and_state_classes_migrator_test.dart [hunk 2]
```diff
@@ -1,40 +0,0 @@
-// Copyright 2020 Workiva Inc.
```
Comment (joebingham-wk) [inline test/boilerplate_suggestors/props_and_state_classes_migrator_test.dart]: This file just got renamed

**Comment by aaronlademann-wf** [comment]
NOTE: This merge to integration includes both CPLAT-9406 and CPLAT-9409 (via the merge of #75)

**Pull request merged**